packages-next/admin-ui/src/pages/ItemPage/index.tsx

```diff
@@ -577,7 +577,7 @@ const ColumnLayout = (props: HTMLAttributes<HTMLDivElement>) => {
   const { spacing } = useTheme();
 
   return (
-    <Container css={{ position: 'relative' }}>
+    <Container css={{ position: 'relative', height: '100%' }}>
```

Comment:
should this be leaving 
```suggestion
    <Container css={{ position: 'relative', height: calc('100%' - 20px) }}>
```
I am not sure how exactly this should be done.

Reply:
Hey, I have the context for this now. Applying `height: 100%` is a fine approach. I'd just add a comment why it's there

Reply:
@gautamsi we don't need to exclude any height there because the container sorts it out